README.md

```diff
@@ -1,2 +1,204 @@
+-
+
+### Usage
+
+- Assuming that you have installed the project's dependencies and started the local server, you can run the project by following these steps:
+
+Open a web browser such as Google Chrome, Mozilla Firefox, or Safari.
+In the address bar, type the URL of the local server that you started in the previous step. This will typically be something like http://localhost:3000.
+Press enter to navigate to the local server's URL.
+The project should now be running in your web browser. Use the website's interface to interact with the project and test its functionality.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+- modules
+
+
+###  Key Features <a name="Key-features"></a>
+- Added index.html file
+- Added styles.css
+- Added index.js file
+-  Used Modular JS
+- React and Redux
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Phelelani Ndlovu**
+
+- GitHub: [@techmoves](https://github.com/techmoves)
+- Twitter: @Dopezonke_
+- LinkedIn: https://www.linkedin.com/in/phelelani-ndlovu-883916160/
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [X] **[ Initialize project with components (React only).]**
+- [x] **[ Add reducers and actions. ]**
+- [X] **[  Use Redux in React components.]**
+- [X] **[ Connect to API. ]**
+- [X] **[ search_currency ]**
+- [X] **[ see_conversion_rate ]**
+- [X] **[ popular_currencies]**
+- [X] **[ Styling]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Understand the project's goals: Before I start contributing to a project, it's important to understand its goals, scope, and target audience. Read the project's documentation and review its codebase to get a sense of what it does and how it works.
+
+Create a new issue: If you I find an existing issue that matches your contribution, consider creating a new one. sure about the problem or feature request clearly and provide any relevant details or context.
+
+
+Contributions, issues, and feature requests are welcome!
+
+
+Feel free to check the [issues page](https://github.com/techmoves/react-capston-project/issues)
+    
+<p align="right">(<a href="">back to top</a>)</p>
+
+<!-- SUPPORT --> 
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+(back to top)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Nelson Sakwa (https://www.behance.net/sakwadesignstudio) for this project inspiration.
+Not forgeting microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] the link to your license is not working as expected, please make sure to fix it before requesting another review.

Reply:
resolved thanks.